src/ContainerAppHelper.ts

```diff
@@ -147,6 +147,37 @@ export class ContainerAppHelper {
         }
     }
 
+    /**
+     * Update container app with update and ingress update to avoid failure of acr authentication.
+     * @param containerAppName - the name of the existing Container App
+     * @param resourceGroup - the resource group that the existing Container App is found in
+     * @param optionalCmdArgs - a set of optional command line arguments
```

Comment:
Can we remove `optionalCmdArgs` on line 154? I'm not seeing it in the method `updateContainerAppIngress` below?

Reply:
makes sense. fixed in the latest commit.